### PR TITLE
When displaying a histogram for a work that displays a price range, highlight the bin representing the middle of the range

### PR DESCRIPTION
**Pull request by ds300**
https://artsyproduct.atlassian.net/browse/PURCHASE-1003

Here's an artwork I rigged up to show this case, so you can test local vs staging https://staging.artsy.net/artwork/alex-katz-diana-6

Local: 
image

Staging:
image

### CODE REVIEW

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit d6d9c3ae3aa8308dadfe4e982cc16e8a5f291f8e

https://deploy-preview-2295--artsy-reaction.netlify.com

File: src/Apps/Artwork/Components/PricingContext.tsx [hunk 1]
```diff
@@ -19,6 +19,10 @@ function PricingContext({ artwork }: PricingContextProps) {
     return null
   }
 
+  const priceCents = artwork.priceCents.max
```
Comment (sweir27) [inline src/Apps/Artwork/Components/PricingContext.tsx]: Do we have to worry about `artwork.priceCents.min` being `null` here as well?

Comment (ds300) [reply src/Apps/Artwork/Components/PricingContext.tsx]: Nope :) If there's no 'min' price then there's no price whatsoever and we don't return histogram data from metaphysics.

**Comment by artsyit** [comment]
:rocket: PR was released in v15.14.7 :rocket:

**Pull request merged**